### PR TITLE
[wip] Allow this bundle to be installed with Symfony3

### PR DESCRIPTION
**Pull request by benji07**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "symfony/framework-bundle": "~2.1",
```
Comment (WouterJ) [inline composer.json]: this needs to have `|~3.0` as well

**Comment by WouterJ** [comment]
Please note that while the code does support Symfony 3, the templates aren't updated for the new profiler and web dev toolbar in Symfony 2.8 and 3.0.

**Comment by WouterJ** [comment]
You should check the `profiler_markup_version` variable like this:
```jinja
{% set profiler_markup_version = profiler_markup_version|default(1) %}

{% if profiler_markup_version == 1 %}
    {# ... markup for the old Profiler #}
{% else %}
    {# ... markup for the new profiler #}
{% endif %}
```

**Comment by TomasVotruba** [comment]
:+1:

**Comment by chandon** [comment]
+1 

With the recent update of symfony 2.8 LTS stable release, i got 9000 deprecation warnings per page because "scope" parameters beeing used by this module.
This PR solve the problem for symfony 2.8, please merge. For the moment, i've to disable Elao/WebProfilerExtraBundle because the 9000 warnings make my dev environment too slow

**Comment by benji07** [comment]
@WouterJ I will create an other PR with the new template

**Pull request merged**